File: fontes/c/micro/micro07.c
#include<stdio.h>

int main()
{
 int programa;
 int numero;
 char opc[5];

 programa = 1;
 do
{
	 printf("Digite um número: ");
	 scanf("%d", &numero);
	 if (numero > 0)
	 {
	 	printf("Positivo\n");
	 }
	 else if (numero == 0)
	 {
	 	printf("O número é igual a 0\n");
	 }
	 else
	 {
	 	printf("Negativo\n");
	 }
	 printf("Deseja finalizar? (sim/nao) ");
	 scanf("%s", opc);
	 if(strcmp (opc, "sim") == 0)
	 {
	 	programa = 0;
	 }
 }
 while(programa == 1);

 return 0;
}

File: fontes/pascal/micro/micro07.pas
program micro07;
 var
 programa: boolean;
 numero : integer;
 opc :char;
 begin
 programa := true;
 while (programa) do
 begin
 write('Digite um numero: ');
 readln(numero);
 if (numero > 0 ) then
 begin
 writeln('o numero eh positivo');
 end
 else
 begin
 if (numero = 0) then
 begin
 writeln('o numero eh 0')
 end
 else
 begin
 writeln('o numero eh negativo');
 end;

 end;
 writeln('Deseja finalizar o programa? R: (S/N) ');
 readln(opc);
 if opc = 'S' then
 begin
 programa := false;
 end
 end;
 end.